content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx

```diff
@@ -164,35 +164,25 @@ Implicit many-to-many relations define relation fields as lists on both sides of
 ```prisma
 model Post {
   id         Int         @id @default(autoincrement())
-  categories Category[]  @relation(references: [id])
+  categories Category[]
 }
 
 model Category {
   id    Int     @id @default(autoincrement())
-  posts Post[]  @relation(references: [id])
+  posts Post[]
 }
 ```
 
-Note that both relation fields in a many-to-many relation must be annotated with the `@relation` attribute and _reference_ the fields of the other model they're mapping to.
+> **Note**: Implicit many-to-many relations do **not** require the `@relation` attribute.
```

Comment:
Mostly yes. Except when you are dealing with ambiguous relations. Then you need to provide the attribute with the name argument.

Reply:
Added link to 'disambiguate relations' and noted that it's not specific to implicit m-n.